tools/osbuilder/Makefile

```diff
@@ -223,6 +225,10 @@ clean:
 	rm -rf $(TARGET_ROOTFS_MARKER) $(TARGET_ROOTFS) $(TARGET_IMAGE) $(TARGET_INITRD) $(DRACUT_OVERLAY_DIR) $(TARGET_IGVM) $(TARGET_IGVM_DEBUG) $(TARGET_IGVM_MSMT) $(TARGET_IGVM_DEBUG_MSMT) $(TARGET_IGVM_LOG)
 	rm -rf $(IGVM_TOOL_SRC)
 
+.PHONY: clean-rootfs
```

Comment:
We definitely want to reduce our changes to this Makefile to the absolute bare minimum. This would be something we should upstream.

Reply:
This is a required piece to allow us to build two images in place. Otherwise, if we do make clean we lose the TARGET_IMAGE from the non-debug build.

Reply:
Three options in preferred order:
- attempt to upstream this change, should be quick
- try your copy step so that we don't need to change this file
- proceed with this as is

Reply:
Decision to proceed as is. Can upstream later. Low risk for hard merge conflicts etc.

Reply:
Proceeding as-is, a change to the makefile without context is hard to justify upstream and the copy step is inelegant side-effect behavior.